### PR TITLE
chore: input errors, use input component in run image

### PR DESCRIPTION
**Pull request by deboer-tim**
### What does this PR do?

Uses the Input component in the Run Image form. This is the first time that we have inputs with 'errors', so I've added support to Input for an error. When there's an error the purple changes to red, there's a red X on the right, and the error message is listed underneath the input.

There is more work to do in this form (e.g. most buttons should probably be Buttons, still a little custom formatting), but I kept this PR to just replacing all the text/file inputs.

### Screenshot / video of UI

Screenshot 2024-02-13 at 4 02 46 PM

### What issues does this PR fix or reference?

Another part of #3234.

### How to test this PR?

Run an image, tab through the form.

### CODE REVIEW

File: packages/renderer/src/lib/ui/Input.svelte [hunk 1]
```diff
@@ -29,45 +30,58 @@ async function onClear() {
...
-      class:hidden="{!value || readonly || disabled}"
-      aria-label="clear"
-      on:click="{onClear}">
-      <Fa icon="{faXmark}" />
-    </button>
+<div class="flex flex-col w-full">
+  <div
+    class="flex flex-row w-full items-center px-1 py-1 group bg-charcoal-500 border-[1px] border-charcoal-500 {$$props.class ||
+      ''}"
+    class:hover:bg-charcoal-900="{!readonly && !disabled}"
```
Comment (benoitf) [inline packages/renderer/src/lib/ui/Input.svelte:37]: It looks like we should find another pattern for specifying classes as I think we reached a state where it's unreadable/unmaintainable
due to all these conditions

**Review by benoitf — APPROVED** [review]

**Review by benoitf — CHANGES_REQUESTED** [review]
it is missing the unit tests for input

File: packages/renderer/src/lib/ui/Input.svelte (hunk 1 above)
Comment (deboer-tim) [reply packages/renderer/src/lib/ui/Input.svelte:37]: Agreed. I was letting it go a bit b/c I didn't know if we still had other requirements, but I think we've got most of the inputs and it's time to simplify.

Any ideas other than extracting a couple variables for the more complicated/merged states? Ok with separate PR?

**Comment by deboer-tim** [comment]
> it is missing the unit tests for input

Doh. Added.

**Review by benoitf — APPROVED** [review]

File: packages/renderer/src/lib/ui/Input.svelte (hunk 1 above)
Comment (benoitf) [reply packages/renderer/src/lib/ui/Input.svelte:37]: Can be follow-up of course, required changes was for the test

**Comment by benoitf** [comment]
FYI a file is not formatted

**Pull request merged**